apps/aecore/src/aec_governance.erl

```diff
@@ -108,8 +109,16 @@ tx_base_gas(oracle_response_tx) -> ?TX_BASE_GAS.
 byte_gas() ->
     ?BYTE_GAS.
 
-minimum_gas_price() ->
+-ifdef(TEST).
```

Comment:
Ok though we should consider remoiving this.
[Tracked](https://www.pivotaltracker.com/story/show/164103197).